docs/crafted-emacs.org

```diff
@@ -144,6 +150,23 @@ for the code examples.
     (straight-use-package 'corfu)
   #+end_src
 
+  You can find a full example to bootstrap straight in the examples directory:
+
+  #+begin_src emacs-lisp
+    (find-file "../examples/crafted-early-init-straight.el")
```

Comment:
```suggestion
    [[file:../examples/crafted-early-init-straight.el][crafted-early-init-straight.el]]
```

This should be a link as this document could be viewed online or in info.  (and therefore don't need the src block tags either. )